### PR TITLE
Add requeue reason to log

### PR DESCRIPTION
**Pull request by kerthcet**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -563,7 +562,7 @@ func (s *Scheduler) requeueAndUpdate(log logr.Logger, ctx context.Context, e ent
 		e.requeueReason = queue.RequeueReasonFailedAfterNomination
 	}
 	added := s.queues.RequeueWorkload(ctx, &e.Info, e.requeueReason)
-	log.V(2).Info("Workload re-queued", "workload", klog.KObj(e.Obj), "clusterQueue", e.ClusterQueue, "queue", klog.KRef(e.Obj.Namespace, e.Obj.Spec.QueueName), "added", added, "status", e.status)
```
Comment (kerthcet) [inline pkg/scheduler/scheduler.go:566]: status is already logged here https://github.com/kubernetes-sigs/kueue/blob/93dce4be3eb2450bbe46d5e1911d208853960625/pkg/scheduler/scheduler.go#L143-L147

Comment (alculquicondor) [reply pkg/scheduler/scheduler.go:566]: `requeueReason`

**Comment by kerthcet** [comment]
Updated.

**Pull request merged**